crates/forge_analyzer/src/utils.rs

```diff
@@ -0,0 +1,15 @@
+use crate::definitions::CalleeRef;
+use swc_core::ecma::ast::{CallExpr, Callee, Expr, MemberProp};
+
+pub fn call_func_with_name(n: &CalleeRef, name: &str) -> bool {
+    if let CalleeRef::Expr(expr) = &n {
+        if let Expr::Member(mem) = &**expr {
+            if let MemberProp::Ident(ident) = &mem.prop {
+                if ident.sym.to_string() == name {
```

Comment:
```suggestion
    return ident.sym == *name;
```
Since `JsWord` impls `PartialEq<str>`. Also, the reason you have to dereference `name` is because `lhs == rhs` in Rust essentially just desugars to `PartialEq::eq(&lhs, &rhs)`. Thus, `ident.sym == name` will try to find an `impl PartialEq<&str>`(which doesn't exist), since it's being passed `PartialEq::eq(&ident.sym, &name)`.